### PR TITLE
Update MethodOverLoadingSample.java

### PR DESCRIPTION
**Pull request by ashan-tharaka**

### CODE REVIEW

File: src/days/day8/MethodOverLoadingSample.java [hunk 1]
```diff
@@ -1,19 +1,31 @@
 package days.day8;
 
-public class MethodOverLoadingSample {
+import java.util.jar.Manifest;
```
Comment (buddhikac96) [inline src/days/day8/MethodOverLoadingSample.java]: Remove this line. We don't need this library here :)

File: src/days/day8/MethodOverLoadingSample.java [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
     }
+}
 
-    float add(int b, int v){
-        return b;
+class Fish extends Animal{
+    void move(){
+        System.out.println("Swim");
     }
 }
```
Comment (buddhikac96) [inline src/days/day8/MethodOverLoadingSample.java]: Add a new line at the end of the code :)
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: src/days/day8/MethodOverLoadingSample.java [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
+import java.util.jar.Manifest;
+
+public class MethodOverloadingSample {
     public static void main(String[] args) {
-        Calculator c = new Calculator();
-        c.add(3);
-        c.add(1, 2);
+
+        Animal a;
+
```
Comment (buddhikac96) [inline src/days/day8/MethodOverLoadingSample.java]: You can add a comment here as well :)

**Review by buddhikac96 — COMMENTED** [review]
Thanks for the code :)
I added some comments. Please go through those and then lets merge this :) Again thanks for the code :)

**Comment by ashan-tharaka** [comment]
updates

**Review by ashan-tharaka — COMMENTED** [review]
updates

**Pull request merged**